cirq/experiments/fidelity_estimation.py

```diff
@@ -0,0 +1,51 @@
+"""Estimate fidelity of large random quantum circuit from observed bitstrings.
+
+Fidelity estimator defined here is used in cross-entropy benchmarking and
+works under the assumption that the evaluated circuit is sufficiently
+random, see https://arxiv.org/abs/1608.00263.
+"""
+
+from typing import cast, List, Sequence, Tuple
+
+import numpy as np
+
+from cirq.circuits import Circuit
+from cirq.ops import Qid
+from cirq.sim import Simulator, WaveFunctionTrialResult
+
+
+def estimate_circuit_fidelity(circuit: Circuit, qubit_order: Sequence[Qid],
+                              bitstrings: Sequence[int]) -> float:
+    """Computes fidelity estimate from one circuit using linear XEB estimator.
+
+    Args:
+        circuit: Random quantum circuit which has been executed on quantum
```

Comment:
Describe in more detail what the conditions on this circuit are. "Random" is not specific.

Reply:
Added comment explaining assumptions. In particular, with regards to circuit randomness, the comment explains that what is needed is the Porter-Thomas output distribution. Also, added a few words about how this assumption can be verified in practice and a reference to a paper with more details.